clients/admin-ui/src/features/common/custom-reports/CustomReportColumn.tsx

```diff
@@ -0,0 +1,4 @@
+export interface CustomReportColumn {
+  label?: string;
+  enabled?: boolean;
+}
```

Comment:
Can this just be moved to a `/src/features/common/custom-reports/types.d.ts` file?